assets/js/blocks/collection-filters/utils.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { ProductCollectionQuery } from '@woocommerce/blocks/product-collection/types';
+
+function taxonomyMapper( taxonomy: string ) {
+	const map: Record< string, string > = {
+		product_tag: 'tag',
+		product_cat: 'cat',
+	};
+
+	return taxonomy in map ? map[ taxonomy ] : `_unstable_tax_${ taxonomy }`;
```

Comment:
How about:

```suggestion
	return map[ taxonomy ] || `_unstable_tax_${ taxonomy }`;
```

Reply:
This is actually my first version, I used `in` in attempt to solve TS error when I didn't type `map`. I forgot to change back. Thanks!